listenbrainz/db/tests/test_user.py

```diff
@@ -19,10 +19,10 @@ def test_update_token(self):
 
     def test_update_last_login(self):
         user = db_user.get_or_create('testlastloginuser')
-        val = int(time.time())
-        db_user.update_last_login(user['musicbrainz_id'], val)
+        db_user.update_last_login(user['musicbrainz_id'])
         user = db_user.get_by_mb_id(user['musicbrainz_id'])
-        self.assertEqual(val, int(user['last_login'].strftime('%s')))
+        # after update_last_login, the val should be equal to the current time
+        self.assertEqual(int(time.time()), int(user['last_login'].strftime('%s')))
```

Comment:
Hmm, I actually thought that this could be a problem later, should have done this from the beginning, done now!